test/old-tests/unit/cuda/test-reduce-randloc.cpp

```diff
@@ -1,463 +0,0 @@
-//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
-// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
-// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
-//
-// SPDX-License-Identifier: (BSD-3-Clause)
-//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
-
-///
-/// Source file containing tests for RAJA GPU tuple min-loc reductions
```

Comment:
@rhornung67 I did not duplicate these tests because most of them are needlessly complex, and their most important functionality is already covered in the forall/indexset tests. Let me know if this is ok with you.